amplify/src/main/java/com/github/stkent/amplify/tracking/rules/VersionNameChangedRule.java

```diff
@@ -36,14 +36,14 @@ public boolean shouldAllowFeedbackPrompt(@NonNull final String cachedEventValue)
     @NonNull
     @Override
     public String getDescription() {
-        return "VersionChangedRule with current app version name " + getCurrentAppVersionName();
+        return "VersionNameChangedRule with current app version name " + getCurrentAppVersionName();
     }
 
     @NonNull
     private String getCurrentAppVersionName() {
         // We access the singleton AppInfoProvider instance statically here to make it possible for
-        // library consumers to create new VersionChangedRule instances easily!
-        return AppInfoProvider.getSharedInstance().getPackageInfo().versionName;
+        // library consumers to create new VersionNameChangedRule instances easily!
+        return AppInfoUtil.getSharedAppInfoProvider().getPackageInfo().versionName;
```

Comment:
This could throw an exception if a library user attempts to configure rules before getting the shared amplify instance. Should probably update so that the constructors for the version rules take a context.